worker/workercmd/worker_nonlinux.go

```diff
@@ -4,14 +4,17 @@ package workercmd
 
 import (
 	"runtime"
+	"time"
 
 	"code.cloudfoundry.org/lager"
 	"github.com/concourse/concourse/atc"
 	"github.com/jessevdk/go-flags"
 	"github.com/tedsuo/ifrit"
 )
 
-type GardenBackend struct{}
+type GardenBackend struct {
```

Comment:
instead of putting them in both worker_linux and worker_nonlinux, can we move this to worker_config. These are where common flags reside.

https://github.com/concourse/concourse/blob/0e1b195bf964cf11e88c6cbd75775f5160cce1df/worker/workercmd/worker.go#L27

Reply:
gooood catch, we indeed thought about that, @kcmannem - we end up putting under `GardenBackend` to make it part of the Garden configuration group 🤔  otherwise, it'd feel kinda out of place (imo)

e.g., `DNSProxyEnable` is under `Garden` too (although yeah, we don't have that for non-linux)